src/common/utxobased/db/Processor.ts

```diff
@@ -368,28 +368,12 @@ export async function makeProcessor(
           txs.push(...txsById)
         }
         if (options != null) {
-          const {
-            startEntries,
-            startIndex,
-            startDate = new Date(0),
-            endDate = new Date()
-          } = options
-
           // Fetch transaction IDs ordered by date
-          let txData: TxIdByDate[]
-          if (startEntries != null && startIndex != null) {
-            txData = await tables.txIdsByDate.queryByCount(
-              '',
-              startEntries,
-              startIndex
-            )
-          } else {
-            txData = await tables.txIdsByDate.query(
-              '',
-              unixTime(startDate.getTime()),
-              unixTime(endDate.getTime())
-            )
-          }
+          const txData: TxIdByDate[] = await tables.txIdsByDate.query(
+            '',
+            0,
+            unixTime(Date.now())
+          )
```

Comment:
How is this change relevant to the upgrade I wonder?

Reply:
My final question now is: is there a new protocol for the core to query the plugin for paging or streaming of this data that we should mark as a TODO?